icu4c/source/i18n/plurrule.cpp

```diff
@@ -397,7 +390,7 @@ getSamplesFromString(const UnicodeString &samples, double *dest,
 
 
 int32_t
-PluralRules::getSamples(const UnicodeString &keyword, double *dest,
+PluralRules::getSamples(const UnicodeString &keyword, FixedDecimal *dest,
```

Comment:
Please also double-check that you didn't redefine public stable API in Java.

Reply:
Done.  The previously-merged Java PR is tagged as internal for CLDR only and private.